docs/scos/dev/developer-getting-started-guide.md

```diff
@@ -160,3 +197,52 @@ Get to know the parts of the Spryker Development Virtual Machine with which we s
 * What is the Spryker DevVM (Development Virtual Machine) and why do we need it?
 * Main Structure
 * Technology Stack: Linux distribution, PHP, Postgres, MySQL, ES, Redis, Queue, Jenkins-->
+
+## 6. CI configuration
+ - [Deployment pipelines](/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/deployment-pipelines.html)
+ - [Customizing deployment pipelines](/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/customizing-deployment-pipelines.html)
+ - [GitHub Actions](/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/configuring-github-actions.html)
+ - [Configuring GitLab pipelines](/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/configuring-gitlab-pipelines.html)
+ - [Azure Pipelines](/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/configuring-azure-pipelines.html)
+ - [Configuring Bitbucket Pipelines ](/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/configuring-bitbucket-pipelines.html)
+
+## 7. Checkers configuration
+### Code sniffer
+Also recommended to update code sniffer to latest version (you also need to change `composer.json` file)
+```bash
+composer update spryker/code-sniffer slevomat/coding-standard --with-dependencies
+```
+
+There are will be some new checks, but mostly everything is easy to fix.
+On a project level you may decide to use your own rules, or exclude something that enamled in spryker by default.
+
+To enable some new rule see full list there: [Slevomat Coding Standard](https://github.com/slevomat/coding-standard)
+
+To disable something you can use configuration. Next example exclude rule that annotation for constructor and method is required:
+```yaml
+<rule ref="vendor/spryker/code-sniffer/Spryker/ruleset.xml">
+        <exclude name="Spryker.Commenting.DocBlock"/>
+</rule>
+```
+### PhpStan
+Recommended update version to 1.2.* (or higher) - it will solve memory issue, and do not have any extra problems.
+
+On a project level you can enable 6 level
+```yaml
+vendor/bin/phpstan analyze -l 6 -c phpstan.neon src/
+```
+
+## 8. PhpStorm configuration
+### Plugins
+ - PYZ
+ - Symfony Support
```

Comment:
please explain what you mean here (lines 236-238) and what specifically should be done at this step.